build_scripts/build_opencv.sh

```diff
@@ -23,6 +23,7 @@ cmake -DCMAKE_BUILD_TYPE=RELEASE \
       -DCMAKE_TOOLCHAIN_FILE=$PWD/../platforms/${OPENCV_TOOLCHAIN_FILE} \
       -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX} \
       -DBUILD_LIST=core,improc,imgcodecs \
+      -DZLIB_ROOT=${INSTALL_PREFIX} \
```

Comment:
I think we should have a separate variable just for that and only use this as a default.
```suggestion
      -DZLIB_ROOT=${ZLIB_ROOT:-$INSTALL_PREFIX} \
```

Reply:
Do you think we should update this inside https://github.com/NVIDIA/DALI_deps/blob/main/build_scripts/build_libtiff.sh#L30 as well?

Reply:
Probably....

Reply:
Done